fews_3di/simulation.py

```diff
@@ -360,17 +373,76 @@ def _add_rain(self, rain_raster_netcdf: Path):
             else:
                 logger.debug("Unknown state: %s", state)
 
+    def _add_constant_rain(self):
+        """Upload constant rainfall and wait for it to be processed."""
+        logger.info("Uploading constant rainfall")
+        duration = self.settings.end - self.settings.start
+        const_rain = openapi_client.models.ConstantRain(
+            simulation=self.simulation_id,
+            offset=0,
+            duration=int(duration.total_seconds()),
+            value=float(self.settings.rain_input),
+            units="m/s",
+        )
+
+        self.simulations_api.simulations_events_rain_constant_create(
+            self.simulation_id, const_rain
+        )
+
+    ## -------------------------------------------------------------------------------------##
+    ## functie voor add_design_rain -- nog niet in api ingebouwd -- kan later nog worden geimplementeerd
```

Comment:
The addition of design rain in the API is still under discussion. Design rain is currently hardcoded in the livesite and added as timeseries rain in the API.